src/test/java/iroha/validation/rules/RulesTest.java

```diff
@@ -0,0 +1,111 @@
+package iroha.validation.rules;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.RETURNS_DEEP_STUBS;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import iroha.protocol.Commands.Command;
+import iroha.protocol.Commands.TransferAsset;
+import iroha.protocol.TransactionOuterClass.Transaction;
+import iroha.validation.rules.impl.SampleRule;
+import iroha.validation.rules.impl.TransferTxVolumeRule;
+import java.math.BigDecimal;
+import java.util.Collections;
+import org.junit.jupiter.api.Test;
+
+class RulesTest {
+
+  private String asset;
+  private Transaction transaction;
+  private TransferAsset transferAsset;
+  private Rule rule;
+
+  private void initTransferTxVolumeTest() {
```

Comment:
You can mark this method with `@Before` annotation if you want to run this method before every test.